playgrounds/react/examples/Examples.tsx

```diff
@@ -0,0 +1,94 @@
+import React from 'react';
```

Comment:
Isn't it the app.tsx file which is moved and renamed? If so, please use git move

Reply:
Hi, I modified the file (moved all examples in a separate folder) and renamed it.